src/main/java/frc/robot/commands/intake/RunIntake.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig;
+import frc.robot.subsystems.intake.Intake;
+
+public class RunIntake extends Command {
+  private final Intake m_intake;
+
+  /**
+   * Creates a new RunIntake command, which runs the roller motor on the intake subsystem to intake
+   * a note
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public RunIntake(Intake subsystem) {
+    m_intake = subsystem;
+
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_intake.run(RobotConfig.IntakeConfig.kDefaultSpeed);
```

Comment:
nit: this should probably be called from `initialize()`, rather than from `execute()` because `execute()` is intended for periodic tasks

Reply:
Done in commit